proj/proj.mk

```diff
@@ -298,6 +298,9 @@ prog: $(CFU_VERILOG)
 bitstream: $(CFU_VERILOG)
 	$(SOC_MK) bitstream
 
+synth: $(CFU_VERILOG)
```

Comment:
does this also need to be implemented for non-hps platforms?

Reply:
@tcal would the regular platform (non-hps) also benefit from this?

Reply:
This particular line (301) (the 'synth' target) is available for all non-sim builds, including regular (common_soc, non-hps) platforms like Arty, Fomu, and Icebreaker.   

If you mean, should we make the entire Vizier framework usable for these other platforms -- it depends on the parameterizations possible with other toolschains.   Yosys has a few options, but they're typically Boolean (on/off), not a numeric parameter.   I don't know offhand if there are similar parameters that we have here with nextpnr-nexus with the other PnR tools -- nextpnr-ice40, nextpnr-ecp5, vpr, or Vivado.  So I guess for each tool, we'd want a table of parameters that can be searched.

There are many parameters to the VexRiscv that could be searched.   @proppy , do you think it would make sense to do that search separately, or combined with the toolchain parameter search?

Reply:
@tcal-x I'd be tempted to have one notebook per parameter "kind" we want to explore (one around pnr, another around risc-v features) so that we can strongly correlate the metrics that we want to score with the parameters we're trying to changes but it'd be also interesting to see how well vizier perform with concurrent parameters exploration.